.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+---
+labels: "Work in Progress"
+name: "Pull Request"
+about: "Regular Pull Request"
+---
+
+#### Pre-Flight checklist
+
+- [ ] Screenshots and design review for any changes that affect layout or styling
+  - [ ] Desktop screenshots
+  - [ ] Mobile width screenshots
+  - [ ] Tag @ferdi or @pdpinch for review
+- [ ] Migrations
```

Comment:
I don't think we need a section about migrations in this, as it's not a Django project.  The settings section below could probably also be removed.  Hugo has a settings file, but it's not the same as a Django settings file and doesn't need devops involvement.